tools/wpt/browser.py

```diff
@@ -49,6 +48,22 @@ def get_ext(filename):
     return ext
 
 
+def get_taskcluster_artifact(index, path):
+    TC_QUEUE_BASE = FIREFOX_CI_ROOT_URL + "/api/queue/v1/"
+    TC_INDEX_BASE = FIREFOX_CI_ROOT_URL + "/api/index/v1/"
+
+    resp = get(TC_INDEX_BASE + "task/%s" % index)
+    resp.raise_for_status()
+    index = resp.json()
+    task_id = index["taskId"]
+
+    resp = get(TC_QUEUE_BASE + "task/%s/artifacts/%s" %
+               (task_id, path))
+    resp.raise_for_status()
```

Comment:
There's no need to do both requests. You request the artifact directly from the index via a URL like this:
```
FIREFOX_CI_ROOT_URL + "/api/index/v1/task/%s/artifacts/%s" % (index, path)
```

Reply:
Woah TIL